### ISSUE TITLE
WARNING: No miners available: disabled by Watchdog

### ISSUE THREAD

**Issue by vstrong-info** [RainbowMiner/RainbowMiner#2752]
I have a few HP DL360/380 servers that I would like to use for CPU mining. None of them have a GPU. CPU mining is enabled. All servers are running in Client mode and report to Server correctly.
Clean Ubuntu 20.04 build.
Additional software: Tailscale client and Docker.
CPU mining is enabled.

What may be a problem?
Thank you.

**Comment by vstrong-info** [comment]
Watchdog is already disabled:
![image](https://github.com/RainbowMiner/RainbowMiner/assets/26736078/f201a049-439a-44b5-954b-d1a829989752)

**Comment by RoyCRoderickIII** [comment]
And your server has a wallet/pool configured  to mine to?

**Comment by vstrong-info** [comment]
Of course. Verus, Zephyr, herominers, viper and luck pool are enabled.

**Comment by RoyCRoderickIII** [comment]
Only thing i can think of off the top of my head is check the console page and see if they are not paused on start.

**Comment by vstrong-info** [comment]
All good but still not working.
![image](https://github.com/RainbowMiner/RainbowMiner/assets/26736078/52fc10ed-9eba-427b-a918-d42f4cf3890c)
![image](https://github.com/RainbowMiner/RainbowMiner/assets/26736078/2a91f64a-7d12-4193-b16c-b59847c591dc)

**Comment by RoyCRoderickIII** [comment]
You may just have to wait for watchdog timer to finish and then should be ok

**Comment by vstrong-info** [comment]
Have been sitting like this for the last 24 hours. Never worked from the moment I installed it. :(

**Comment by RoyCRoderickIII** [comment]
Try running repair.sh and restart something ist right, ive only ever seen it do that if some dependencies or something was missing etc

**Comment by vstrong-info** [comment]
Thank you. I tried running repair.sh but unfortunately it did not help. I did not install any GPU drivers as I do not have any GPUs installed in this system.

**Comment by RoyCRoderickIII** [comment]
You should not need to, did you make sure to enable cpu mining? During setup cpu mining is disabled be default. That is the only other reason i can think of cpus wouldn't mine

**Comment by vstrong-info** [comment]
I did enable CPU mining during setup as that was the whole purpose of using those servers. 
I take Ubuntu 20.04 is fully supported, right?

**Comment by RoyCRoderickIII** [comment]
Yes ofcourse you may need to create a debug  file at the bottom of the menu on the gui, cause if sll you say is true something else is wrong

**Comment by vstrong-info** [comment]
[debug_2024-04-29.zip](https://github.com/RainbowMiner/RainbowMiner/files/15148989/debug_2024-04-29.zip)
Would you be able to check it out please? Thank you

**Comment by RoyCRoderickIII** [comment]
okay I am by far an expert on RBM, but the only thing i see that catches my attention because i have never seen it before is here in the startup

"Detecting devices ..
1 CPU found
Initialize configuration ..
Re-installing libraries and binaries ..
Checking last shutdown ..
crashed
Read configuration ..
Starting sysinfo ..
ok"
i have never on any of my CPU only systems seen it try and reinstall libraries or binaries. if repair did not work, maybe try update.sh? something is definitely off . i know on the arm versions we have to run the install script directly, could give that a try

sudo pwsh  ./Scripts/Install.ps1
thats from the RainbowMiner directory.

**Comment by vstrong-info** [comment]
Thank you but unfortunately it did not help. :(
[debug_2024-04-29-2.zip](https://github.com/RainbowMiner/RainbowMiner/files/15152166/debug_2024-04-29-2.zip)

**Comment by RainbowMiner** [comment]
You seem to have made a mistake with your wallets inside the pools.config.txt - could you please send me your current pools.config.txt file to info@rbminer.net so that I can have a look? 
At the moment, it looks like you have setup your ZEPH wallet address as a coin symbol. It should be "ZEPH" as coin symbol and "$ZEPH" as wallet address.

**Comment by whitindllc** [comment]
i see this as an issue
     "ZEPHYR35YYc6wkhNBcMuxP6aWoQeV7SC8bBr7ytKmSRAc9eXcqWzC7xAqkKH1yz18CJUpuB2jsHwCh4fSQDdD8V1EZXfDzk22433u": "",
      "ZEPHYR35YYc6wkhNBcMuxP6aWoQeV7SC8bBr7ytKmSRAc9eXcqWzC7xAqkKH1yz18CJUpuB2jsHwCh4fSQDdD8V1EZXfDzk22433u-Params": "",
 

should look like this

    "PYI": "pyrin:blahblahblah",
    "PYI-Params": "",
    "ZEPH": "zepBlahblahblah",
    "ZEPH-Params": "",  

i stopped there.... fix them and it should fix the issue

**Comment by whitindllc** [comment]
what payments do you have to ZEPHYR35YYc6wkhNBcMuxP6aWoQeV7SC8bBr7ytKmSRAc9eXcqWzC7xAqkKH1yz18CJUpuB2jsHwCh4fSQDdD8V1EZXfDzk22433u
